homeassistant/components/trafikverket_train/sensor.py

```diff
@@ -117,24 +128,32 @@ def next_departuredate(departure):
 class TrainSensor(SensorEntity):
     """Contains data about a train depature."""
 
-    _attr_device_class = SensorDeviceClass.TIMESTAMP
-
-    def __init__(self, train_api, name, from_station, to_station, weekday, time):
+    def __init__(
```

Comment:
You can set icon and device_class at the class level.
```suggestion
    _attr_icon = ICON
    _attr_device_class = SensorDeviceClass.TIMESTAMP
    
    def __init__(
```

Reply:
I will of course change. Is there a point here except "visual"? Not a pro so nice to know. 😄

Reply:
I'll let a "pro" confirm...